### DISCUSSION TITLE
Race map: The Virtual Land (WIP).

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1158] [Maps and Content Showcase]
A race map that is set into a retrowave-inspired world.
It features a very stylized, yet simple artstyle, with lots of bright colors around the map, used as identification most of the time.

Red: Harder, but faster route/ Death.
Blue: Easy, but longer route. Not too much deaths await for you.
Green: Checkpoints/the path outside.

Right now it takes about ~~ 3 minutes to be finished.
It has some custom neon-trees and a simple texture created for this map, along with a "moving" red plane of death.
Since this is in W.I.P. state, expect things to change... like the tower part, which still isn't 100% decided yet on it's layout.

The neon-forest is using LOD, so it shouldn't have too much of a impact on the performance, which i believe will only be impacted by the use of some volumetric lights around the map.

![20190210215655](https://user-images.githubusercontent.com/24359960/52542122-b01a7380-2d83-11e9-9578-d8c38485ad30.png)
![20190210215702](https://user-images.githubusercontent.com/24359960/52542116-a5f87500-2d83-11e9-8ff5-f5df1a18a5d5.png)
![20190210215711](https://user-images.githubusercontent.com/24359960/52542117-a8f36580-2d83-11e9-8ea7-a42c3df4c2df.png)
![20190210215735](https://user-images.githubusercontent.com/24359960/52542126-b6a8eb00-2d83-11e9-9189-30ac6e2fe379.png)
![20190210215739](https://user-images.githubusercontent.com/24359960/52542131-ba3c7200-2d83-11e9-90bd-454798e68329.png)
![20190210215834](https://user-images.githubusercontent.com/24359960/52542135-bdcff900-2d83-11e9-87e6-a3be304d326f.png)
![20190210220033](https://user-images.githubusercontent.com/24359960/52542136-c3c5da00-2d83-11e9-81bb-4b5d74bf6265.png)

Any feedback is welcome.
[retrowave-WIP.zip](https://github.com/red-eclipse/world/files/2851100/retrowave-WIP.zip)  
Updated because i forgot to place the texture in it's correct location.

**Comment by lambda-11235** [comment]
How are you supposed to make this jump?
![20190210212039](https://user-images.githubusercontent.com/16354815/52547116-d5a27f80-2d79-11e9-99d7-b9840c0a4a5c.png)

**Comment by SniperGoth** [comment]
You climb above you, then jump. There's a plattform up there.
Also, i updated the zip because i... forgot to place the texture where it should had been.
